charts/argocd-apps/values-integration.yaml

```diff
@@ -833,3 +833,70 @@ applications:
           secretKeyRef:
             name: smartanswers-zendesk-client
             key: password
+- name: feedback
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/feedback
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        valueFrom:
+          secretKeyRef:
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: SUPPORT_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-support-bearer-feedback
+            key: bearer_token  
+      - name: SUPPORT_API_BEARER_TOKEN
+        valueFrom: 
+          secretKeyRef:
+            name: signon-token-support-api-bearer-token-feedback
+            key: bearer_token
+      - name: PUBLISHING_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-publishing-api-bearer-feedback
+            key: bearer_token
+      - name: ASSISTED_DIGITAL_GOOGLE_SPREADSHEET_KEY
+        valueFrom: 
+          secretKeyRef:
+            name: assisted-digital-google-spreadsheet-key-feedback
+            key: bearer_token
+      - name: GOOGLE_CLIENT_EMAIL
+        valueFrom: 
+          secretKeyRef:
+            name: google-client-email-feedback
+            key: bearer_token
+      - name: GOOGLE_PRIVATE_KEY
+        valueFrom: 
+          secretKeyRef:
+            name: google-private-key-feedback
+            key: bearer_token
```

Comment:
GOOGLE_CLIENT_EMAIL and GOOGLE_PRIVATE_KEY are a username/password pair and should therefore be stored as two keys in the same AWS/Kubernetes secret, for example:

```json
{
  "username": "foo@bar.com",
  "password": "hunter2"
}
```

Reply:
This is so that we're not making it unnecessarily difficult for someone else to implement automatic rotation of these creds in future.

Reply:
Hi Chris, I'll amend that accordingly, many thanks.

Reply:
Amended